docs/01-start-here/01-installation-steps.md

```diff
@@ -53,15 +63,15 @@ You need a Mac or Linux PC (Ubuntu).
     $ cd frontend
     ```
 
-1. Get AWS Credentials using [Janus](https://janus.gutools.co.uk/) for *Frontend* and *CMS fronts* (you will need access to Janus)
+1. Get AWS Credentials using [Janus](https://janus.gutools.co.uk/) for *Frontend*, *CMS fronts* and *CAPI* (You will need access to Janus).
```

Comment:
Do you mean Content API for CAPI? In Janus to get the credential it's called `Content API`

Reply:
Yeah, I probably did. Sorry I was quoting from bad memory...